src/AutomatedCar/Models/AutomatedCar.cs

```diff
@@ -1,25 +1,65 @@
 namespace AutomatedCar.Models
 {
+    using Avalonia;
     using Avalonia.Media;
+    using global::AutomatedCar.SystemComponents.Packets;
+    using ReactiveUI;
+    using System;
+    using System.Threading;
     using SystemComponents;
-
-    public class AutomatedCar : Car
+    public class AutomatedCar : Car, IControlledCar
     {
         private VirtualFunctionBus virtualFunctionBus;
 
+        public VirtualFunctionBus VirtualFunctionBus { get => this.virtualFunctionBus; }
```

Comment:
Why do we need this second virtual function bus?

Reply:
Az első egy adattag, a második egy tulajdonság (kis v vs. nagy V), utóbbi az MVVM architektúra miatt kell. Ha a kettő összevonható, akkor az a C#-képzetlenségemből ered.